redirector.k8s.io/OWNERS

```diff
@@ -0,0 +1,9 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- thockin
+- cblecker
+
+# limiting factor from being an approver is inability to deploy changes once PRs merge
```

Comment:
If we deploy this to existing clusters, yes.  If we assume this does not launch until new cluster is up, I don't see why you could be approver

Reply:
> Why would this not be "go.k8s.io" dir, since that is the name we have already?

Cause the redirector can do more than just serve go.k8s.io. Therefore making it a service distinct from the first use case aka go.k8s.io seemed like a good solution.

Reply:
Not sure if I understood this correctly. Do you mean I should remove the reviewer role and add myself to approver for simplification assuming this will only deployed to the new cluster?
(went ahead and did this change)